Neos.Media.Browser/Classes/Controller/AssetController.php

```diff
@@ -468,6 +475,32 @@ public function variantsAction(string $assetSourceIdentifier, string $assetProxy
         }
     }
 
+    /**
+     * (Re-)create all variants for the given image
```

Comment:
Did you intend to add an option to force recreation of asset variants with this action or is this just a missleading comment?